### PR TITLE
Allow users to browse top rated movies

### PR DESCRIPTION
**Pull request by sted1994**
## What does this PR do?
Creates a carousel that allows users to look through all movies with a rating higher than 6.5.

## Where should the reviewer start?
browser

## How should this be manually tested?
Run npm test

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "@testing-library/user-event": "^13.5.0",
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
+    "react-responsive-carousel": "^3.2.23",
```
Comment (emmacbd) [inline package.json:11]: neat!

File: src/Carousel.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+<p>Top rated</p>
+    <div className="carousel">
+
+      {featureCards}
+    </div>
+    </div>
+              )
+}
+
+export default FeatureCarousel
```
Comment (emmacbd) [inline src/Carousel.js:25]: good call on making this a separate component.

File: src/App.js [hunk 3]
```diff
@@ -47,7 +50,7 @@ class App extends Component {
     return (
       <main>
         < Nav returnHome={this.returnHome}/>
-        {(this.state.errors) ? < Error /> : (this.state.movieSummary) ? < MovieSummary movieToView={this.state.movieToView}/> : < PosterContainer movies={this.state.movies} showMovieSummary={this.showMovieSummary}/>}
+        {(this.state.errors) ? < Error /> : (this.state.movieSummary) ? < MovieSummary movieToView={this.state.movieToView}/> : < HomePage moviesProp={this.state.movies} showMovieSummary={this.showMovieSummary}/>}
```
Comment (michaelputnam67) [inline src/App.js:53]: This may be to complex of a ternary perhaps this could be refactored to be easier to read.

**Review by michaelputnam67 — COMMENTED** [review]
Great job figuring out this carousel Tyler!

File: src/Carousel.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+  const featureCards = featureMovies.map(movie => {
+    return < CarouselCard showMovieSummary={showMovieSummary} movieProp={movie}  id={movie.id} key={movie.id}/>
+  })
+
+console.log()
+  return (
+<div className="carousel-display">
+<p>Top rated</p>
+    <div className="carousel">
```
Comment (emmacbd) [inline src/Carousel.js:17]: maybe add the actual rating beneath the card?

**Review by emmacbd — COMMENTED** [review]
Nice job on the carousel!

**Pull request merged**